getting_started/getting_started.rst

```diff
@@ -209,6 +209,8 @@ Home can be accessed through ssh utilities:
 
       ssh userid@home.ccs.ornl.gov
 
+For more information on connecting to OLCF resources, see the <.connecting>`__ section.
```

Comment:
I just noticed that we have two duplicated sections "Systems Available to All Projects" and "Connecting for the first time" in both the "Getting Started" and "Connecting" sections. It would be good to reconcile these somehow so we don't have duplicated content that can drift out of sync. Probably one should refer back to the other?